tests-invoke

```diff
@@ -252,8 +256,21 @@ class PullTask(object):
             cmd = ["timeout", "120m", entry_point]
             if opts.verbose:
                 cmd.append("--verbose")
-            ret = subprocess.call(cmd)
-
+            # Popen instead of call; while not etc... (SubProcess.poll)
+            if ret:
```

Comment:
This looks out of place -- this is the "then" branch of `if not ret:`, and it's not reassigned anywhere in between. Probably this whole if is just an overzealous copy&paste?